onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -1054,15 +1056,20 @@ CUDAExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph,
       // cast is not compute heavy, and may be placed outside
     }
 
+
     if (!not_supported && !force_inside) {
       // Note that nodes with only inputs from initializer would not be place on CUDA
       // Ideally, those nodes should be eliminated in constant folding
       bool all_non_initializer_inputs_from_outside = true;
```

Comment:
seems the comment and variable name are no longer accurate. Maybe change to should_force_outside?